src/theme/Theme.jsx

```diff
@@ -53,9 +52,8 @@ function loadTheme(mode, setMode, themeChangeListeners) {
   // https://mui.com/customization/dark-mode/
   const activePalette = mode === Themes.Day ? day : night
   const theme = {
-    components: getComponentOverrides(activePalette, getTypography()),
-    typography: getTypography(),
-    shape: {borderRadius: 8},
+    components: getComponentOverrides(activePalette),
+    shape: {borderRadius: 0},
```

Comment:
In your previous PR I question removing border radius.. let's figure that out and do whatever we decide here